### PR TITLE
Firstendpoint

### PR DESCRIPTION
**Pull request by NikitaSelanko**
finished first endpoint, added MVC test, changed gitignore

### CODE REVIEW

**Comment by Dutton1990** [comment]
Your test is just checking that it's returning an array at the minute, it could be an empty array. Either amend this test or write another to check that you have a certain amount of categories back (how many do you expect?) and that each category has a slug and a description :) 

Once you've done that and pushed to this branch again just send another 'nchelp pr' with this link again and we will check it 😄

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((result) => {
+          const catArr = result.body.categories;
+          expect(catArr).toBeInstanceOf(Array);
+          expect(catArr.length).toBe(4)
+          catArr.forEach((category)=>{
+            expect(category.hasOwnProperty("slug")).toBe(true)
+            expect(category.hasOwnProperty("description")).toBe(true)
```
Comment (alexdevdavis) [inline __tests__/app.test.js:31]: test could be strengthened by checking the data type of the values too

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          })
+        });
+    });
+    test("status 200, should responde withan array of category objects, each of which should have the following properties: slug & description properties", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((result) => {
+          const catArr = result.body.categories;
+          console.log(catArr, "TEST <<<<<<<<<<<<<<<");
```
Comment (alexdevdavis) [inline __tests__/app.test.js]: remove floaty logs

File: __tests__/app.test.js (hunk 1 above)
Comment (alexdevdavis) [reply __tests__/app.test.js:31]: ok, I now see you've done this in the below test :)

**Review by alexdevdavis — APPROVED** [review]
Happy to approve this now; tests are much more robust. Good job. 
 Please remove floaty console.logs.

**Pull request merged**